java/client/src/main/java/glide/api/models/commands/ListDirection.java

```diff
@@ -0,0 +1,19 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.ListBaseCommands;
+
+/**
+ * Enumeration representing element popping or adding direction for the {@link
+ * ListBaseCommands#lmpop} command.
```

Comment:
Add refs to bzmpop and lmove

Reply:
I feel like we should just ref the ListBaseCommand class like: 
```suggestion
/**
 * Enumeration representing element popping or adding direction for the {@link
 * ListBaseCommands} commands.
```
since more commands might be added that again need to be added here, like `blmove`